### PR TITLE
Adding documentation for prospect bulk export updated_at procedure

### PR DESCRIPTION
**Pull request by marcus-moncayo**

### CODE REVIEW

File: docs/kb/api-version-3/export.md [hunk 1]
```diff
@@ -79,6 +79,46 @@ OR
...
+* **updated_before**: (Optional) Selects prospects that were created before the specified time. This value must be after the value in `updated_after`. If this argument is not specified, then no upper boundary is used in the query, and all data after the `updated_after` is returned. The value can be `today`, `yesterday`, `last_7_days`, `this_month`, `last_month`, or a custom time specified in [GNU Date Input Syntax](http://www.gnu.org/software/tar/manual/html_node/Date-input-formats.html) format.
+* **deleted**: (Optional) Includes deleted prospects. When this field is set to `true`, only deleted prospects are returned. When this field is set to `false`, only prospects that are not deleted are returned.  When this field is set to `all`, both deleted and not deleted prospects are returned. The values can be `true`,`false`, or `all`. If this argument is not specified only prospects that are not deleted will be returned.
```
Comment (nihunt) [inline docs/kb/api-version-3/export.md:118]: Due to there being an issue with an index for isArchived we are currently not supporting `true` and `false` params correct.

Here was the [PR](https://git.dev.pardot.com/Pardot/pardot/pull/34844/files) where you changed to only support `all`.

I checked [latest master](https://git.dev.pardot.com/Pardot/pardot/blob/master/apps/frontend/lib/moduleManagers/api/export/procedures/ProspectLastActivityProcedure.php#L66-L78) and it seems like we are still restricting to only `all`.

File: docs/kb/api-version-4/export.md [hunk 2]
```diff
@@ -79,6 +79,46 @@ OR
...
+* **updated_before**: (Optional) Selects prospects that were created before the specified time. This value must be after the value in `updated_after`. If this argument is not specified, then no upper boundary is used in the query, and all data after the `updated_after` is returned. The value can be `today`, `yesterday`, `last_7_days`, `this_month`, `last_month`, or a custom time specified in [GNU Date Input Syntax](http://www.gnu.org/software/tar/manual/html_node/Date-input-formats.html) format.
+* **deleted**: (Optional) Includes deleted prospects. When this field is set to `true`, only deleted prospects are returned. When this field is set to `false`, only prospects that are not deleted are returned.  When this field is set to `all`, both deleted and not deleted prospects are returned. The values can be `true`,`false`, or `all`. If this argument is not specified only prospects that are not deleted will be returned.
```
Comment (nihunt) [inline docs/kb/api-version-4/export.md:118]: Same as my comment above.  We should update to currently only support `all`.

File: docs/kb/api-version-3/export.md [hunk 3]
```diff
@@ -79,6 +79,46 @@ OR
...
+
+__NOTE:__ The range between created_after and created_before cannot exceed 1 year. When created_before is not specified, the current date is used to gauge the interval.
```
Comment (nihunt) [inline docs/kb/api-version-3/export.md:120]: For the `filter_by_updated_at` we are only looking at update_at correct?  You should change to `updated_after` and `updated_before`

File: docs/kb/api-version-3/export.md (hunk 1 above)
Comment (marcus-moncayo) [reply docs/kb/api-version-3/export.md:118]: 🤦‍♂ good catch will update

File: docs/kb/api-version-3/export.md [hunk 4]
```diff
@@ -79,6 +79,46 @@ OR
 
 __NOTE:__ The range between created_after and created_before cannot exceed 1 year. When created_before is not specified, the current date is used to gauge the interval.
 
+## Prospect
+
+### Fields
+
+Select the prospect fields that need to be exported. [Following](../../object-field-references/#prospect) are the fields that are available for prospect. The value for `fields` must be an array of strings of the available fields.
```
Comment (nihunt) [inline docs/kb/api-version-3/export.md:86]: Looks like this just points to the [list of fields available from sync api](http://developer.pardot.com/kb/object-field-references/#prospect)?  I know the sync api does some joins with other tables and [not all the fields there are available in export](https://git.dev.pardot.com/Pardot/pardot/blob/master/test/integration/frontend/api/ApiExportRequestManagerIntegrationTest.php#L47-L93) - `notes` and `score` come to mind.

Comment (marcus-moncayo) [reply docs/kb/api-version-3/export.md:86]: Yeah I may need to follow up with PAN on what to do here.

File: docs/kb/api-version-3/export.md [hunk 5]
```diff
@@ -79,6 +79,46 @@ OR
 
 __NOTE:__ The range between created_after and created_before cannot exceed 1 year. When created_before is not specified, the current date is used to gauge the interval.
 
+## Prospect
+
+### Fields
+
+Select the prospect fields that need to be exported. [Following](../../object-field-references/#prospect) are the fields that are available for prospect. The value for `fields` must be an array of strings of the available fields. 
+
+### filter_by_updated_at
```
Comment (nihunt) [inline docs/kb/api-version-3/export.md:88]: Do we care about documenting `last_activity_at` filter?

Comment (marcus-moncayo) [reply docs/kb/api-version-3/export.md:88]: We did not implement the required deleted param. I think we should wait until we have the proper indexes on [last activity at](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000080n9aIAA/view). It is also not a general use case. @jpearson-sfdc  thoughts? I would tag Chris Cornett but I do not see a tag for him.

File: docs/kb/api-version-3/export.md (hunk 4 above)
Comment (marcus-moncayo) [reply docs/kb/api-version-3/export.md:86]: @jpearson-sfdc  As nick noted above the fields for the sync api do not match up. Do you have a preference on how we should handle this?

Comment (marcus-moncayo) [reply docs/kb/api-version-3/export.md:86]: Should we denote using a column on the object field references page?

**Comment by gupta298** [comment]
Can you change https://github.com/pardot/api-docs/blob/7bea992ba5dd565feec58cd877c94b54cc960a98/mkdocs.yml#L48 as well?
Also, has this text been approved by Casey or Lorie? Wondering if some of this can be generalized, like abilities and date range being within 1 year, etc.

**Comment by marcus-moncayo** [comment]
@gupta298 I saw that removed in your pr so I didn't remove it here.

**Comment by gupta298** [comment]
Ahh I see, maybe you should do it as well depending on if this will be merged before my PR.

**Pull request closed without merging**